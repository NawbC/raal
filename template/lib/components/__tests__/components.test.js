import React from 'react';
import { shallow, mount } from 'enzyme';
// import renderer from 'react-test-renderer';
import Components from '../components';


describe('A suite', function () {

	it('should be selectable by class "muguet"', function () {
		expect(shallow(<Components />).is('.muguet')).toBe(true);
	});

	it('should mount in a full DOM', function () {
		expect(mount(<Components />).find('.muguet').length).toBe(1);
	});

	it('Snapshot', () => {
		// const component = renderer.create(<Components />);
		// const componentInstance = testRenderer.root;
		// testInstance.findByProps({ id: 'click1-button' }).props.onClick();
		// testInstance.findByProps({ className: 'clickText' })

		// let snapshot = component.toJSON();
		// expect(snapshot).toMatchSnapshot();
		// snapshot.props.onClick();
		// snapshot = component.toJSON();
		// expect(snapshot).toMatchSnapshot();
	});
});


