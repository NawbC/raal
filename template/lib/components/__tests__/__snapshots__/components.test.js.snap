// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`A suite Snapshot 1`] = `
<div
  className="muguet"
  onClick={[Function]}
>
  
   create-muguet-lib
</div>
`;

exports[`A suite Snapshot 2`] = `
<div
  className="muguet"
  onClick={[Function]}
>
  <span>
    loveÂ 
  </span>
   create-muguet-lib
</div>
`;
